cmd/scollector/collectors/processes_windows.go

```diff
@@ -40,7 +40,7 @@ func c_windows_processes() (opentsdb.MultiDataPoint, error) {
 	}
 
 	var svc_dst []Win32_Service
-	var svc_q = wmi.CreateQuery(&svc_dst, `WHERE Started=true`)
+	var svc_q = wmi.CreateQuery(&svc_dst, "")
```

Comment:
@alienth It reflects the Object (svc_dst) and creates a WMI sql like query. You can specify a where filter as a string.